csharp/ql/test/experimental/CWE-918/RequestForgery.cs

```diff
@@ -0,0 +1,61 @@
+// semmle-extractor-options: /r:System.Text.RegularExpressions.dll /r:HtmlAgilityPack.dll /r:Microsoft.Extensions.Configuration.dll /r:System.Net.Http.dll /r:Microsoft.Extensions.Logging.dll /r:System.Threading.Tasks.dll /r:Microsoft.AspNetCore.Mvc.dll
```

Comment:
@porcupineyhairs I think you are simply missing `/r:System.Private.Uri.dll` from this line.